libs/dev-dock/backend/src/dev_dock_api.ts

```diff
@@ -50,7 +56,10 @@ const MOCK_CARD_SCRIPT_PATH = join(
   '../../../auth/scripts/mock-card'
 );
 
+// Create a stable directory for dev-dock data
+const DEV_DOCK_DIR = join(homedir(), '.vx-dev-dock');
 export const DEV_DOCK_FILE_PATH = '/tmp/dev-dock.json';
+const DEV_DOCK_ELECTION_PATH = join(DEV_DOCK_DIR, 'election.json');
```

Comment:
Maybe the dev dock JSON file should live in the same place as the dev dock election?